docs/tests.md

```diff
@@ -6,6 +6,9 @@ describes them and give pointers on how to run/debug them.
 These tests are described in the order in which we prefer to write them. Always
 prefer unit tests, integration tests, and system tests, in that order.
 
+Ensure that you have set a default dev hub, through one of the `sfdx force:auth:*` commands,
```

Comment:
@GaryFNC, @guw, @Sweetman, @DatGuyJonathan - Take note of this. This means that you need to have a default scratch org set authenticated against (doesn't need to be production) if you want to run *all* tests. We are starting with the sobject tests, but eventually as more integration tests are written, those will be affected as well.

Reply:
@chowwinston -  Can you actually put the actual command and parameters that you use here? That would help standardize what we use. And we can also compare this to what the appveyor and travis files are doing.